client-sdk/go/modules/contracts/types.go

```diff
@@ -139,6 +139,18 @@ type CodeQuery struct {
 	ID CodeID `json:"id"`
 }
 
+// CodeStorageQuery is the body of the contracts.CodeStorage query.
+type CodeStorageQuery struct {
+	// ID is the code identifier.
+	ID CodeID `json:"id"`
+}
+
+// CodeStorageQueryResult is the result of the contracts.CodeStorage query.
+type CodeStorageQueryResult struct {
+	// Value is the storage value or nil if code ID doesn't exist.
```

Comment:
```suggestion
	// Value is the contract code.
```
Maybe the field should just be renamed to `code` or something? Value seems too generic.